modules/core/04-channel/keeper/upgrade.go

```diff
@@ -191,7 +191,7 @@ func (k Keeper) WriteUpgradeTryChannel(ctx sdk.Context, portID, channelID string
 	previousState := channel.State
 	channel.State = types.TRYUPGRADE
 	// TODO: determine flush status
-	// channel.FlushStatus = flushStatus
+	channel.FlushStatus = types.FLUSHING
```

Comment:
currently setting to flushing as default for now, as flush status must be in flushing or flush complete to proceed within the handler tests.